go/vt/vttablet/tabletserver/vstreamer/vstreamer.go

```diff
@@ -23,6 +23,8 @@ import (
 	"io"
 	"time"
 
+	"google.golang.org/protobuf/proto"
+
 	"vitess.io/vitess/go/vt/vttablet"
 
 	"google.golang.org/protobuf/encoding/prototext"
```

Comment:
It's very nitty, I know, but can you please arrange it so that the imports are in the following groups and order:
<builtin packages>

<external/3rd party packages>

<vitess.io packages>